package.json

```diff
@@ -9,7 +9,20 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "css-loader": "^5.0.2",
+    "style-loader": "^2.0.0",
     "webpack": "^5.21.2",
     "webpack-cli": "^4.5.0"
+  },
+  "dependencies": {
+    "bootstrap": "^4.6.0",
+    "eslint": "^7.19.0",
```

Comment:
- [ ] Kindly add the necessary start scripts in package.json:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

Reply:
- If you would be including the start script section in the script then you would need to add the webpack-dev-server package also.